src/Orleans/Messaging/SocketManager.cs

```diff
@@ -65,9 +67,16 @@ private Socket SendingSocketCreator(IPEndPoint target)
                 s.NoDelay = true;
                 WriteConnectionPreemble(s, Constants.SiloDirectConnectionId); // Identifies this client as a direct silo-to-silo socket
                 // Start an asynch receive off of the socket to detect closure
-                var foo = new byte[4];
-                s.BeginReceive(foo, 0, 1, SocketFlags.None, ReceiveCallback,
```

Comment:
I have no concerns, just wasn't sure what we were getting from this change.  It sounds like the port was only of Orleans core.  That's good.  Don't need to port everything at once. :)
We should discuss possible replacement of net network layer in relation to CoreCLR port as if we're replacing it, there is less reason to port all of the networking code.